server/http.go

```diff
@@ -518,11 +518,12 @@ func handleTokenFunc(srv OIDCServer) http.HandlerFunc {
 			}
 		case oauth2.GrantTypeRefreshToken:
 			token := r.PostForm.Get("refresh_token")
+			scopes := r.PostForm.Get("scope")
```

Comment:
If the users omit the "scope" field, it's the same as requesting the scopes present in the original requests.[0] How does the code handle this case?

I'm basically looking for some code that looks like:

```
if len(scopes) == 0 {
    scopes = record.Scopes
}
```

Though I can't find it.

I also don't see a test case[1] that tests for that case.

[0] https://tools.ietf.org/html/rfc6749#section-6
[1] https://github.com/bobbyrullo/dex/blob/aa3f0681a9666941b9fa5e9e8ce579715bcb039c/server/server_test.go#L487